packages/react/src/components/Breadcrumb/BreadcrumbItem.js

```diff
@@ -67,7 +67,7 @@ const BreadcrumbItem = React.forwardRef(function BreadcrumbItem(
     <li className={className} ref={ref} {...rest}>
       {React.cloneElement(children, {
         'aria-current': ariaCurrent,
-        className: `${prefix}--link`,
+        className: cx(`${prefix}--link`, children?.props?.className),
```

Comment:
```suggestion
        className: cx(`${prefix}--link`, children.props.className),
```

Should be good to go with `children.props` always being defined! Let me know if that's not the case 👀 

I believe if it's not an element then `React.cloneElement` would fail

Reply:
Good point :+1: